frontend/express/public/javascripts/countly/vue/core.js

```diff
@@ -129,7 +155,27 @@
                         if (appObj) {
                             context.commit("setActiveApp", Object.freeze(JSON.parse(JSON.stringify(appObj))));
                         }
+                    },
+                    removeActiveApp: function(context) {
+                        context.commit("setActiveApp", null);
```

Comment:
you should aslo remove the app from local store here.
`store.remove('countly_active_app');`

Reply:
incase you missed this :)